joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -368,7 +557,16 @@ class AstCreator(filename: String, global: Global)
             astForExpressionContext(exp)
           })
           .toSeq
-      Ast().withChildren(expAsts)
+      val callNode = NewCall()
+        .name(Operators.arrayInitializer)
+        .methodFullName(Operators.arrayInitializer)
+        .signature(Operators.arrayInitializer)
+        .typeFullName(MethodFullNames.UnknownFullName)
+        .dispatchType(DispatchTypes.STATIC_DISPATCH)
+        .code(ctx.getText)
+        .lineNumber(-1)
```

Comment:
Please do not set `lineNumber` to -1. If you don't want to set it, just don't set it.

Reply:
Done